zap/scripts/authentication/get_bearer_token.js

```diff
@@ -52,19 +52,19 @@ function authenticate(helper, paramsValues, credentials) {
     signer.update([headers, token].join(".").getBytes(StandardCharsets.UTF_8));
 
     const signature = encoder.encodeToString(signer.sign());
-    const jwt = headers + "." + token + "." + signature;
+    const jwt = `${headers}.${token}.${signature}`;
```

Comment:
It's probably worth doing a zap scan against this branch to make it's okay with these changes.

Reply:
I did run a zap scan, I added a screenshot in the description. Link to the scan: https://dev.azure.com/NHSD-APIM/API%20Platform/_build/results?buildId=235972&view=results

Reply:
Ah yeah, sorry about that :D